Packs/TruSTAR/Integrations/TruSTAR_V2/TruSTAR_V2.py

```diff
@@ -632,6 +660,44 @@ def search_reports(self, search_term=None, enclave_ids=None):
         entry = self.get_entry(title, reports, ec)
         return entry
 
+    def redact_report(
+            self,
+            title=None,
+            report_body=None,
+            enclave_ids=None,
+            external_url=None,
+            time_began=None,
+            distribution_type="ENCLAVE"):
+        """
+        Redacts and submits a new report to TruSTAR station.
+
+        :param title: Title of the report.
+        :param report_body: Body of the report.
+        :param enclave_ids: Enclave IDs where to submit the report.
+        :param external_url: External URL of the report.
+        :param time_began: Incident time. Defaults to current time if not given.
+        :param distribution_type: Whether the report will be in the community, or only
+        in enclaves
+
+        :return: Entry context with the submitted report.
+        """
+        response = self.client.redact_report(
```

Comment:
There are 2 different functionalities here. In some cases we want to let the user submit a report with the literal title and body and in others they can 'redact' a report. This means that some terms that they have on their redaction library will be applied, so those terms on the redaction library will be replaced by a generic tag. In this particular command when a user redacts a report, it will also be submitted with the redaction terms applied.

You can see an example on our API docs here: https://docs.trustar.co/api/v13/reports/redact_report.html